docs/install/command-line.md

```diff
@@ -1,52 +1,101 @@
+title: Kubo on the command line
+description: Using IPFS Kubo through the command-line allows you to do everything that IPFS Desktop can do, but at a more granular level, since you can specify which commands to run. Learn how to install it here.
 current-ipfs-version: v0.17.0
 ---
 
-# Command-line
+# Install Kubo on the command line
 
-Installing IPFS through the command-line is handy if you plan on building applications and services on top of an IPFS node. This method is also useful if you're setting up a node without a user interface, usually the case with remote servers or virtual machines. Using IPFS through the command-line allows you to do everything that IPFS Desktop can do, but at a more granular level since you can specify which commands to run.
+This guide describes the available installation processes for IPFS Kubo, a Go-based implementation of the IPFS protocol. Kubo was the first implementation of the IPFS, and is the most widely used implementation today. Kubo allows you to do everything that IPFS Desktop can do, but at a more granular level, since you can specify which commands to run. Kubo has the following features:
 
-![A terminal window running the IPFS daemon in Ubuntu.](./images/command-line/wsl-running-ipfs-in-linux.png)
+- An IPFS daemon server
+- Extensive command line tooling
+- An HTTP RPC API for controlling the node
+- An HTTP Gateway for serving content to HTTP browsers
+- Binaries for Windows, MacOS, Linux, FreeBSD and OpenBSD
+
+Installing Kubo in the command line is handy for many use cases, such as building applications and services on top of an IPFS node, or setting up a node without a user interface (which is usually the case with remote servers or virtual machines).  
+
+To get started, familiarize yourself with the system requirements. Then, determine if you'd like to install Kubo using one of the 5 official binary distributions, or build Kubo from source. Once you've installed Kubo, determine which node to use in the command line. Finally, check out the next steps.
+
+> :warning:
+> Usually, installing Kubo using one of the official binaries is the appropriate option. Building from source is only recommended if you are running Kubo on a system with severe resource constraints, or are contributing to the Kubo project.
 
 ## System requirements
 
-Kubo IPFS requires 512MiB of memory and can run an IPFS node on a Raspberry Pi. However, how much disk space your IPFS installation takes up depends on how much data you're sharing. A base installation takes up about 12MB of disk space. One can enable automatic garbage collection via [--enable-gc](/reference/kubo/cli/#ipfs-daemon) and adjust the [default maximum disk storage](https://github.com/ipfs/kubo/blob/v0.17.0/docs/config.md#datastorestoragemax) for data retrieved from other peers.
+IPFS runs on most Windows, MacOS, Linux, FreeBSD and OpenBSD systems. Ensure that your system meets the following minimum requirements:
+
+- 512MiB of memory
```

Comment:
512 or 2GB? 

(this line or the one below?)

Reply:
🤦 whoops lol @2color good catch.

I'm actually not sure. @lidel would you know the answer to this one? https://docs.ipfs.tech/install/command-line/#system-requirements says 512, https://github.com/ipfs/kubo/blob/v0.17.0/README.md#system-requirements says 2

Reply:
If we mean default settings, I was unable to run it with 1GB. 
I'd say update docs to say 2GB.

Reply:
ah sorry @lidel just saw this, I was following the Slack convo

Reply:
Slack thread https://filecoinproject.slack.com/archives/C03FFEVK30F/p1670435968403349

Reply:
Updated in https://github.com/ipfs/ipfs-docs/pull/1342/commits/7679f3df8a11e25778805ead2af27e745b8609a1